src/clojupyter/misc/display.clj

```diff
@@ -1,70 +1,68 @@
 (ns clojupyter.misc.display
   (:require [clojupyter.protocol.mime-convertible :as mc]
             [clojupyter.misc.states :as states]
-            [cheshire.core :as cheshire]
-            [clojure.data.codec.base64 :as b64]
-            [clojure.java.io :as io])
-  (:import [javax.imageio ImageIO]))
+            [hiccup.core :as hiccup]))
 
 (defn display [obj]
   (swap! (:display-queue @states/current-global-states) conj (mc/to-mime obj))
   nil)
 
-;; Incanter Plot
 
-(defrecord IncanterPlot [chart width height])
+;; Html
 
-(extend-protocol mc/PMimeConvertible
-  IncanterPlot
-  (to-mime [plot]
-    (let [out (io/java.io.ByteArrayOutputStream.)
-          {:keys [chart width height]} plot]
-      (ImageIO/write (.createBufferedImage chart width height)
-                     "png" out)
-      (mc/stream-to-string
-       {:image/png (str (apply str (map char (b64/encode (.toByteArray out)))))}))))
+(defrecord HtmlString [html]
 
-(defn make-incanter-plot [chart & {:keys [width height]
-                                   :or {width 600 height 400}}]
-  (IncanterPlot. chart width height))
+  mc/PMimeConvertible
+  (to-mime [_]
+    (mc/stream-to-string
+     {:text/html html})))
 
+(defn html [html-str]
+  (HtmlString. html-str))
 
-;; Html
+(defn ^:deprecated make-html
```

Comment:
I do not see why `make-html` is depreciated. Surly it is less practical than `hiccup-html` but still totally valid and useful.

Reply:
Oh I added `html` as an alias effectively. The `make-*` seemed redundant so the behavior of `display/html` will work exactly as `display/make-html` will

Reply:
I may actually make it so there's only `display/html` and it converts to hiccup on vectors and leaves as is for string. Might be a lot of magic, but I think we want a really simple API for people.

Reply:
I agree at making `display/html` magic.